src/components/tool/guide/Instructions/StartInstructions.js

```diff
@@ -34,7 +35,7 @@ export function LyraStartInstructions({
             In the ssh session, run the following command to schedule the {jobType.toLowerCase()} job:
         </InstructionText>
         <CopyBox>
-            {`qsub${jobType === "Interactive" ? " -I" : ""} -l walltime=1:00:00 -l ${resources.join(":")}`}
+            {`qsub${jobType === "Interactive" ? " -I" : ""} -l walltime=${wallTime?.hour}:${wallTime?.minute}:00 -l ${resources.join(":")}`}
```

Comment:
Should be left-pad with leading `0`s when it's a single digit

Reply:
Fixed